airbyte-integrations/template/singer-source/README.md

```diff
@@ -1,4 +1,35 @@
-# Quick manual
+### Create an Airbyte Source from a Singer Tap
+1. If you are trying to create an Airbyte Source from scratch (not using a Singer Tap), you should follow [this guide](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/template/python-source/README.md) instead.
+1. Copy the template into a directory with your integration name. `cp -r airbyte-integrations/template/singer-source airbyte-integrations/singer/SOURCENAME`.
+1. ```
+   mv airbyte-integrations/singer/SOURCENAME/template_singer_source airbyte-integrations/singer/SOURCENAME/SOURCENAME_singer_source
+   ``` 
+1. Update the name, description, and dependencies in `setup.py`
+    1. Your Singer Tap's package should be added as a dependency.
+    1. Note that release dependencies should go in `setup.py`, NOT `requirements.txt`. The requirements file is only used for local development. 
+1. Update the `Dockerfile` to point `CODE_PATH` and `AIRBYTE_IMPL_MODULE` at `SOURCENAME_singer_source`
+1. Edit the classname in `SOURCENAME_singer_source/source.py` and update the `Dockerfile` to point `AIRBYTE_IMPL_PATH` to that classname. 
+1. Update the `io.airbyte.name` label in the `Dockerfile` to `airbyte/source-SOURCENAME-singer`
+1. Define a specification for the configuration when connecting to the source by modifying `SOURCENAME_singer_source/spec.json`.
+    1. This spec uses JsonSchema to define properties and validations.
+    1. Ultimately, configurations that match this schema will be generated by the Airbyte UI and passed to your integration as input. You can also manually generate config files and pass them to the integration CLI.
+1. `SOURCENAME_singer_source/source.py` is where you actually implement your integration.
+    1. `check` should check if a connection can be established to a source given a config that matches your specification.
+    1. `discover_cmd` returns a command string we will execute to retrieve a Singer catalog.
+        1. Behind the scenes, we convert this Singer catalog to an `AirbyteCatalog`. 
+        1. In some cases, your Singer Tap might not implement discover. In this case, you will have to add functionality to create your own catalog.
+    1. `read_cmd` will write messages to stdout. 
+        1. The command to specify is typically the same as the sync from a Singer Tap.
+        1. Usually this command needs to be computed from the config file and optionally the state.
+1. Update the `sample_files` directory with an example config and catalog (discover output).
```

Comment:
why is this necessary?

Reply:
I'm adding this under the `README.md` request as
```
    1. We recommend including sample configs and catalogs in the `sample_files` directory or within your `README.md`.
```

It's not necessary, just recommended so it's easy to work with.